apps/ledger-live-mobile/src/components/FirmwareUpdateBanner.tsx

```diff
@@ -40,15 +49,81 @@ const FirmwareUpdateBanner = ({
     hasCompletedOnboardingSelector,
   );
 
-  const [showDrawer, setShowDrawer] = useState<boolean>(false);
+  const [staxBattery, setStaxBattery] = useState<
+    { percentage: number; charging: boolean } | undefined
+  >();
+  const [showBatteryWarningDrawer, setShowBatteryWarningDrawer] =
+    useState<boolean>(false);
+
+  useEffect(() => {
+    if (lastConnectedDevice?.modelId === DeviceModelId.stax) {
+      const sub = concat(
+        withDevice(lastConnectedDevice.deviceId)(transport =>
+          from(
+            getBatteryStatus(transport, BatteryStatusTypes.BATTERY_PERCENTAGE),
+          ),
+        ),
+        withDevice(lastConnectedDevice.deviceId)(transport =>
+          from(getBatteryStatus(transport, BatteryStatusTypes.BATTERY_FLAGS)),
+        ),
```

Comment:
Considering we will most likely need the tuple of information `battery %, isCharging` you should extend the command to accommodate this. Make it do the two apdus, get the combined information in one line and not this mess.